Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+edition = "2021"
+
+name = "BlackThorn"
+version = "0.1.0"
+description = "A free, simple tool for creating database diagrams"
```

Comment:
How about `a tool for creating database diagrams and migration scripts` ?

Reply:
Will think about the proper README and documentation during the release.